components/Footer.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-footer class="footer">
```

Comment:
Instead of class footer we can wrap inside's of it in vuetify's grid system this will prevent extra css
```html
<v-row justify="center" align="center">
  <v-col cols="3" md="2" class="text-center pr-4">
```